semeio/jobs/scripts/overburden_timeshift.py

```diff
@@ -38,14 +39,13 @@ def _get_args_parser():
     return parser
 
 
-def main_entry_point(args=None):
+def main_entry_point():
     parser = _get_args_parser()
-    options = parser.parse_args(args)
+    options = parser.parse_args()
     if options.help_params:
         print(generate_rst_doc())
     else:
-        ots_run(options.config)
-
-
-if __name__ == "__main__":
```

Comment:
Removing the `if __name__...` I would agree on!

Reply:
I will move the console scripts to another PR